test/EVMHost.h

```diff
@@ -104,21 +114,21 @@ class EVMHost: public evmc::Host
 		evmc::bytes32 const& _value
 	) noexcept;
 
-	evmc::uint256be get_balance(evmc::address const& _addr) noexcept final
+	evmc::uint256be get_balance(evmc::address const& _addr) const noexcept final
 	{
 		if (Account const* acc = account(_addr))
```

Comment:
Well, given @axic's comment https://github.com/ethereum/solidity/pull/7848/files#r352180437, we could just return 1 here for ``_addr < 1024``, couldn't we?